lawlibrary/templates/peachjam/_footer.html

```diff
@@ -50,6 +50,8 @@ <h5>{% trans 'Our partners' %}</h5>
 
   <h6 class="mt-4">{% trans 'With support from' %}</h6>
   <div><a href="https://www.uct.ac.za/" target="_blank" rel="noreferrer">{% trans 'University of Cape Town' %}</a></div>
+  <h6 class="mt-4">{% trans 'Content licensing' %}</h6>
```

Comment:
Content licensing is a sub-issue of Terms of Use. Instead, I suggest putting a simply "Terms of Use" link beneath the main "About" block, or in a new row beneath all the columns -- effectively it stands alone.

Reply:
done